### ISSUE TITLE
Bundle-audit on current codebase 

### ISSUE THREAD

**Issue by ang-st** [openware/peatio#337]
Hi,
this one check for vulnerable dependencies :

```

ubuntu@ip-xxx:~/rubykube/peatio$ bundle audit check --update
Updating ruby-advisory-db ...
Cloning into '/home/ubuntu/.local/share/ruby-advisory-db'...
remote: Counting objects: 3523, done.
remote: Compressing objects: 100% (21/21), done.
remote: Total 3523 (delta 17), reused 31 (delta 16), pack-reused 3484
Receiving objects: 100% (3523/3523), 596.07 KiB | 0 bytes/s, done.
Resolving deltas: 100% (1727/1727), done.
Checking connectivity... done.
Updated ruby-advisory-db
ruby-advisory-db: 293 advisories
Name: rack-attack
Version: 3.0.0
Advisory: 132234
Criticality: Unknown
URL: https://github.com/kickstarter/rack-attack/releases/tag/v4.3.1
Title: rack-attack Gem for Ruby missing normalization before request path
processing
Solution: upgrade to >= 4.3.1

Name: rest-client
Version: 1.6.9
Advisory: CVE-2015-3448
Criticality: Unknown
URL: http://www.osvdb.org/show/osvdb/117461
Title: Rest-Client Gem for Ruby logs password information in plaintext
Solution: upgrade to >= 1.7.3

Name: rest-client
Version: 1.6.9
Advisory: CVE-2015-1820
Criticality: Unknown
URL: https://github.com/rest-client/rest-client/issues/369
Title: rubygem-rest-client: session fixation vulnerability via Set-Cookie headers in 30x redirection responses
Solution: upgrade to >= 1.8.0

Vulnerabilities found!
```

Best

**Comment by yivo** [comment]
Hi @ang-st 

Thank you for your submit. We will validate all cases.

**Comment by yivo** [comment]
@vpetrusenko Please, work in devel branch.

**Comment by yivo** [comment]
Fixed in #348.
